backend/routes/flightStatus.js

```diff
@@ -0,0 +1,44 @@
+import { API_KEY, API_SECRET } from "../config.js";
+import Amadeus from "amadeus";
+import express from "express";
+import { PrismaClient } from '@prisma/client'; 
+
+const prisma = new PrismaClient();
+const router = express.Router();
+const amadeus = new Amadeus({
+    clientId: API_KEY,
+    clientSecret: API_SECRET,
+});
+
+//flight search for adding trip
+router.get('/flight-status', async (req, res) => {
+    try {
```

Comment:
We need to verify the user session is valid here and throw an error in case it is not.  How did you do that in codepath lab?